packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerForm.tsx

```diff
@@ -90,6 +193,10 @@ const SchedulerForm: FC<
     const isAddSlackDisabled = disabled || slackState !== SlackStates.SUCCESS;
     const isAddEmailDisabled = disabled || !health.data?.hasEmailClient;
 
+    const [isCsv, setIsCsv] = useState<boolean>(
+        methods.getValues()?.format === 'csv',
+    );
```

Comment:
This is causing issues when we update a scheduler. 
I suggest we use the `const format = methods.watch('format');` instead of using the internal state.

Reply:
Yep, nice suggestion. Done